### DISCUSSION TITLE
Dabigatran Reversal PBPK Model

### DISCUSSION THREAD

**Discussion by LMFuhr** [Open-Systems-Pharmacology/Forum#730] [Announcement]
Dear OSP-community,

We have recently been working on a mechanistic PBPK/PD model to study dabigatran reversal, which includes the reversal of dabigatran by hemodialysis and its antidote idarucizumab.

Therefore, a PBPK model of idarucizumab has been carefully developed and evaluated. The model was then coupled with a recently published PBPK model of dabigatran to predict the interaction of both compounds. The dabigatran model was furthermore extended by a dialyzer compartment and a hemodialysis clearance as well as by a simple PD model.

Related publications:
Fuhr LM, Hanke N, Meibohm B, Lehr T
[Effective Removal of Dabigatran by Idarucizumab or Hemodialysis: A Physiologically Based Pharmacokinetic Modeling Analysis](https://link.springer.com/article/10.1007/s40262-019-00857-y) Clinical Pharmacokinetics 2020

All models can be found [here](https://github.com/Open-Systems-Pharmacology?utf8=%E2%9C%93&q=antidote&type=&language=).

Best regards,
Laura

**Comment by tobiasK2001** [comment]
Congratulation! Very nice work!
